packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1052,8 +1052,60 @@ export function isBStackSession(config: Options.Testrunner) {
     return false
 }
 
-export function shouldAddServiceVersion(config: Options.Testrunner, testObservability?: boolean): boolean {
-    if (config.services && config.services.toString().includes('chromedriver') && testObservability !== false) {
+export function isBrowserstackInfra(config: BrowserstackConfig & Options.Testrunner, caps?: Capabilities.BrowserStackCapabilities): boolean {
```

Comment:
can we add some code comments in function, helps what that if or else is supposed to look for

Reply:
Okay

Reply:
@xxshubhamxx don't see changes for this